src/framework.cpp

```diff
@@ -44,16 +44,17 @@
 #define MAX_FARKAS_CST 2000
 
 // This value represents the upper bound on w in case the dependence is short.
-#define SHORT_DEP_DISTANCE 10
+#define MAX_SHORT_DEP_DISTANCE 10
```

Comment:
Pls update the comment above to make it independent on 'w' - for eg. it's the threshold on something to be treated as a short dependence for ... purposes.